definitions/firefox_desktop.toml

```diff
@@ -1115,6 +1115,15 @@ description = """
 Count of impressions of Sponsored Pocket content.
 """
 
+[metrics.sponsored_pocket_ctr]
```

Comment:
For experiments this will compute the average over clients of the client-level CTRs. Is that desired?

Reply:
No that is not ideal, is there any way to have it calculated at the day level?  Or is this just a fundamental part of how jetstream/etc works?

Reply:
This might be a separate question by where do I set the analysis period?  Is it strictly a documentation thing?

Reply:
It sounds like you'd want to make a change to make the metric a "ratio" metric where the numerator and denominator are summed independently. To do that you would define the numerator and denominator separately as metrics then define the metadata to make the ratio ([example)](https://github.com/mozilla/metric-hub/blob/main/definitions/firefox_desktop.toml#L3-L12). Then you give that ratio metric the `population_ratio` statistic and specific the numerator and denominator ([example](https://github.com/mozilla/metric-hub/blob/main/jetstream/outcomes/firefox_desktop/firefox_suggest.toml#L12-L14)). 

Analysis periods are explicitly specified for custom metrics or for defaults. For metrics in Outcomes, they're automatically set to `overall` and `weekly`. Does this absolutely have to be daily?

Reply:
It'd be a little more costly to compute, querying data every day in addition to 1x/week. However, more urgently is the issue that outcomes are currently only weekly and overall. So this would have to be changed somehow, or this metric is added as a default to all experiments. 

Is this maybe something better included on a monitoring dashboard for all experiments than as a statistical output for some of them? I'm trying to understand the use case here

Reply:
In that case I think weekly is fine

Reply:
In that case I think weekly is fine, it's not so important we need to urgently add daily to outcomes or something